lib/sqlalchemy/sql/ddl.py

```diff
@@ -496,7 +491,6 @@ def __init__(
         quote=None,
         cascade=False,
         if_exists=False,
-        if_not_exists=False,
```

Comment:
**mike bayer** (zzzeek) wrote:

same here

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4067